### PR TITLE
cpu/stm32_common: fix PLLs enabling and usage

### PR DESCRIPTION
**Pull request by vincent-d**
This is a small follow up of #7477. I I realized I needed R factor for PLL I2S on stm32f2 to use as MCO2.

So this add possibility to use R output for every PLL.

### CODE REVIEW

**Comment by vincent-d** [comment]
Found another bug with extra PLL enabling, fixed it.

File: cpu/stm32_common/stmclk.c [hunk 1]
```diff
@@ -47,6 +47,16 @@
 #define PLL_SRC                  RCC_PLLCFGR_PLLSRC_HSI
 #endif
 
+#if defined(CPU_FAM_STM32F2)
+#define RCC_PLLCFGR_PLLP_Pos    (16U)
```
Comment (aabadie) [inline cpu/stm32_common/stmclk.c]: Small nit: the defines could be aligned with the ones below

Comment (vincent-d) [reply cpu/stm32_common/stmclk.c]: Fixed and amended directly

**Comment by aabadie** [comment]
Needs rebase here

**Comment by vincent-d** [comment]
Rebased

**Comment by aabadie** [comment]
and go

**Pull request merged**